lib/schema/schema.go

```diff
@@ -674,7 +674,7 @@ const manifestSchema = `
           "type": "object",
           "additionalProperties": false,
           "properties": {
-            "hairpinMode": {"enum": ["promiscuous-bridge", "hairpin-veth"], "default": "promiscuous-bridge"},
+            "hairpinMode": {"enum": ["promiscuous-bridge", "hairpin-veth"], "default": ""},
```

Comment:
Yea, I went this route because:
- The fields itself is part of the external API, so don't want to break any tele builds where the field happens to be set. Ideally I would be able to comment that the field is deprecated and to be removed in the next version of the object.
- IIRC the default broke some unit tests that expect the default against the removed constant. Since the field is effectively ignored now, it seemed reasonable to just change the default to match the unit tests vs. leave the constant / unit tests.